department-of-veterans-affairs/va.gov-cms
# VACMS-6071: Create three new VAMC Top Task content types.

## Description

Relates to #6071
## Testing done
Visual, phpunit, Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/129643332-2f04e0aa-927e-4a4a-8055-9f568690625c.png)
![image](https://user-images.githubusercontent.com/2404547/129643354-012cf210-5785-40a4-8d08-d68bd310244c.png)
![image](https://user-images.githubusercontent.com/2404547/129643369-189af2e1-7e78-48a7-814c-ce20413eedb1.png)

## QA steps
- As an administrator (or content_admin), create 1 node here: `/node/add/vamc_system_medical_records_offi` & here `/node/add/vamc_system_register_for_care` & here: `/node/add/vamc_system_billing_insurance`
- Prior to save, visually verify the pages match the screenshots above
- After save, visually verify that title is `Register for care` or `Medical records office` or `Billing and insurance`
- Login as ryan.thurlwell@va.gov and confirm that you cannot view any of the three new node add forms
- Visit all three of the new nodes created by admin, and confirm that you can edit and save each of the new nodes.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
